packages/ibm-products-styles/src/components/Tearsheet/_tearsheet_next.scss

```diff
@@ -0,0 +1,364 @@
+/**
+ * Copyright IBM Corp. 2024, 2025
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+@use '@carbon/styles/scss/theme' as *;
+@use '@carbon/styles/scss/colors' as *;
+@use '@carbon/styles/scss/spacing' as *;
+@use '@carbon/styles/scss/motion' as *;
+@use '@carbon/styles/scss/breakpoint' as *;
+@use '@carbon/styles/scss/type';
+@use '@carbon/styles/scss/utilities';
+@use '../../global/styles/project-settings' as *;
+@use '../../global/styles/mixins' as *;
+
+$block-class__next: #{$pkg-prefix}--tearsheet__next;
+$motion-duration: $duration-moderate-02;
+
+/// Experimental tearsheet styles
+@include block-wrap($block-class__next) {
+  // ──────────────────────────────────────────────────────────────
+  // Container
+  // ──────────────────────────────────────────────────────────────
+  .#{$block-class__next}__container {
+    align-self: flex-end;
+    block-size: 100%;
+    inset-block-start: auto;
+    max-block-size: calc(100% - #{$spacing-09});
+    transform: translate3d(0, calc(min(95vh, 500px)), 0);
```

Comment:
The placement of the tearsheet should be pinned to the bottom of the window, there currently is a gap at the bottom.

Reply:
Nmv, this appears to be fixed now 🎉